content/blog/2024-11-elixir-exchange/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "How I Learned to Wrap Tools from the Galaxy EU Team"
+date: "2024-11-01"
+authors: "Marisa Loach"
+tease: "An ELIXIR Staff Exchange to expand the single cell tools on Galaxy"
+---
+I am currently a PhD student at The Open University in the UK, where I reanalyse public single cell RNA sequencing data using Galaxy to explore how cells change in disease. However, I recently took a break from my project to learn how to add more tools to the Galaxy platform. As [my poster at GCC2024](https://f1000research.com/posters/13-806) explained, I went from being a user who wished for more tools to being a user-developer who can wrap those tools for herself.
+
+<p align="center">
+  <img src="./rocket-marisa.png" alt="A rocket with the Use Galaxy logo carries three users, one with a thought bubble reading I wish Galaxy had this tool... and the others with speech bubbles saying We need more tools for integration and annotation and Where is the latest version of Seurat. Two developers look on from behind their computer screens at a desk with the Galaxy Europe logo on it." width="400"/>
+</p>
+
+I was able to do this thanks to the **ELIXIR Staff Exchange Programme**, which enabled me to spend six months working with the Galaxy EU team in Freiburg. The exchange gave me a chance to see the parts of Galaxy that users don’t usually encounter. It was amazing to see how hard the team works to keep Galaxy running and I was particularly impressed by how attentive they were to the Galaxy help forums and Matrix channels.
+
+## Wrapping Seurat for Galaxy
+
+During the exchange I learned how tools are incorporated into Galaxy through wrappers, which set out the tool requirements, the options that users will see on the interface, and the commands that Galaxy will perform when someone hits the Run Tool button. I used what I learned to wrap the latest version of the Seurat pipeline, which I was already familiar with thanks to my focus on single cell omics during my PhD. 
+
+I drew on my experience of using Galaxy to reorganise the Seurat tools, grouping them together by theme, so that we now have wrappers for preprocessing, dimensional reduction, data integration, and other activities. Each of these wrappers contains a selection of related functions, including some that weren’t previously available on Galaxy. Once the tools were working, I also wrote a tutorial for the Galaxy Training Network (GTN) to help people start using them for their own analyses. 
+
+### The New Seurat Toolsuite
+
+- **Seurat Create:** load transcriptomic or CITEseq data into a Seurat Object, add QC metrics, and filter the dataset
```

Comment:
You can add links to the tools if you like or even buttons like here https://github.com/galaxyproject/galaxy-hub/blob/e54dc35cd16ef7accc4218dd41bedb8cd7c2fa78/content/news/2022-02-24-alphafold/index.md?plain=1#L46